bika/lims/browser/analysisrequest/analysisrequests.py

```diff
@@ -80,744 +86,531 @@ def __init__(self, context, request):
+                    t(_("Assigned")), self.portal_url),
+                "contentFilter": {
+                    "assigned_state": "assigned",
+                    "cancellation_state": "active",
+                    "review_state": ("sample_received",
+                                     "attachment_due",),
+                    "sort_on": "created",
+                    "sort_order": "descending",
+                },
+                "transitions": [
+                    {"id": "receive"},
+                    {"id": "retract"},
+                    {"id": "prepublish"},
+                    {"id": "cancel"},
+                ],
+                "custom_transitions": [print_stickers],
+                "columns": self.columns.keys(),
+            }, {
+                "id": "unassigned",
+                "title": "<img title='%s' src='%s/++resource++bika.lims.images/unassigned.png'/>" % (
+                    t(_("Unassigned")), self.portal_url),
+                "contentFilter": {
+                    "assigned_state": "unassigned",
+                    "cancellation_state": "active",
+                    "review_state": (
+                        "sample_received",
+                        "attachment_due",
+                    ),
+                    "sort_on": "created",
+                    "sort_order": "descending",
+                },
+                "transitions": [
+                    {"id": "receive"},
+                    {"id": "retract"},
+                    {"id": "prepublish"},
+                    {"id": "cancel"},
+                ],
+                "custom_transitions": [print_stickers],
+                "columns": self.columns.keys(),
+            },
+        ]
+
+    def update(self):
+        """Called before the listing renders
+        """
+        super(AnalysisRequestsView, self).update()
+
+        self.workflow = api.get_tool("portal_workflow")
+        self.member = self.mtool.getAuthenticatedMember()
+        self.roles = self.member.getRoles()
+
+        setup = api.get_bika_setup()
+
+        # remove `to_be_sampled` filter
+        if not setup.getSamplingWorkflowEnabled():
+            self.review_states = filter(
+                lambda x: x.get("id") != "to_be_sampled", self.review_states)
+
+        # remove `scheduled_sampling` filter
+        if not setup.getScheduleSamplingEnabled():
+            self.review_states = filter(
+                lambda x: x.get("id") != "scheduled_sampling", self.review_states)
+
+        # remove `to_be_preserved` filter
+        if not setup.getSamplePreservationEnabled():
+            self.review_states = filter(
+                lambda x: x.get("id") != "to_be_preserved", self.review_states)
+
+        # remove `rejected` filter
+        if not setup.getRejectionReasons():
+            self.review_states = filter(
+                lambda x: x.get("id") != "rejected", self.review_states)
```

Comment:
Thanks for removing these filters, but remember this should be done in a separate PR :P